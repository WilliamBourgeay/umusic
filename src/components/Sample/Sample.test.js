import React from 'react';
import renderer from 'react-test-renderer';

import Sample from './Sample';

it('should renders without ui change', () => {
  const component = renderer.create(<Sample name="sample" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
