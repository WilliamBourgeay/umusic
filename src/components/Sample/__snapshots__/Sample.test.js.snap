// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders without ui change 1`] = `
<button
  type="button"
>
  sample
  bite
</button>
`;
